### DISCUSSION TITLE
Precomputed slider server: sliding over custom vector

### DISCUSSION THREAD

**Discussion by VarLad** [JuliaPluto/PlutoSliderServer.jl#44] [Q&A] [answered]
I've a few questions regarding https://github.com/JuliaPluto/PlutoSliderServer.jl/pull/29

Firstly, can I use it to do something like the following, when exported to static HTML?

https://user-images.githubusercontent.com/51269425/145789463-df810cac-63e9-43ca-bb34-ba1e997a5824.mp4

**Comment by fonsp** [comment, marked as answer]
That's a funny application, and yes! this will work! Please post your other questions too, this is really useful feedback :)

**Comment by fonsp** [comment]
Hey @VarLad ! Let me know if you have more questions! I think it would be best to open a new discussion for each question
